Packs/Vectra/Integrations/Vectra_v2/Vectra_v2.py

```diff
@@ -141,7 +141,9 @@ def fetch_incidents(self, last_run: Dict):
             'query_string': query_string,
             'page_size': self.fetch_size,
             'page': 1,
+            'order_field': 'last_timestamp'
         }
+        demisto.debug(f'\n\nparams: {params}\n\n')
```

Comment:
If we want to leave this here let's remove the newline chars and write a better message